### ISSUE TITLE
TimeoutOverflowWarning on Upload

### ISSUE THREAD

**Issue by MohammedAl-Mahdawi** [velocityzen/dropbox-stream#3]
Hi,

I'm trying to upload a movie that have more than 300MB size and here is my code:

```
const fs = require('fs');
const path = require('path');
const accessToken = 'MY TOKEN IS HERE'
const db = require('dropbox-stream');

const FILETOUPLOAD = '/path/to/movie.flv';

let up = db.createDropboxUploadStream({
    token: accessToken,
    filepath: '/test/' + path.basename(FILETOUPLOAD),
    chunkSize: 1000 * 1024,
    autorename: true
  })
  .on('done', res => console.log('Success', res))
  .on('progress', res => console.log(res))
  .on('error', err => console.log(err))

fs.createReadStream(FILETOUPLOAD).pipe(up);
```
However my console fills in this message:

```
(node:16533) TimeoutOverflowWarning: 4294967296000 does not fit into a 32-bit signed integer.
Timer duration was truncated to 2147483647.
34603008
```

As you can see here:

![selection_007](https://user-images.githubusercontent.com/5562839/34356307-ba8c8a8a-ea4d-11e7-91cb-d5bf8742dde2.jpg)

The upload process failed and here is the end of the console:

```
(node:16533) TimeoutOverflowWarning: 4294967296000 does not fit into a 32-bit signed integer.
Timer duration was truncated to 2147483647.
309329920
_http_outgoing.js:506
    throw new errors.TypeError('ERR_INVALID_CHAR', 'header content', name);
    ^

TypeError [ERR_INVALID_CHAR]: Invalid character in header content ["Dropbox-API-Arg"]
    at validateHeader (_http_outgoing.js:506:11)
    at ClientRequest.setHeader (_http_outgoing.js:510:3)
    at new ClientRequest (_http_client.js:173:14)
    at Object.request (https.js:245:10)
    at Req._send (/path/to/app/node_modules/hyperquest/index.js:137:21)
    at /path/to/app/node_modules/hyperquest/index.js:51:21
    at process._tickCallback (internal/process/next_tick.js:150:11)
```

Can you please tell me how to solve this?

All the best,
Mohammed

**Comment by MohammedAl-Mahdawi** [comment]
The video name was in Arabic language and contained "+18" characters, after renaming it to small letter English name it uploaded successfully but this message:

```
(node:16533) TimeoutOverflowWarning: 4294967296000 does not fit into a 32-bit signed integer.
Timer duration was truncated to 2147483647.
```

Still appear and fill my console.

**Comment by velocityzen** [comment]
I'll check with the characters. Seem like a bug. About the timeout message, please update all the nodejs and dependecies to the latest and check if it will happen again.
Thank you

**Comment by MohammedAl-Mahdawi** [comment]
Hi @velocityzen 

Thank you for the reply.

I've updated everything and restarted my computer and the issue still there.
Here my system info:

```
OS: Linux Mint 18.2 Cinnamon
Node: v9.3.0
npm: 5.6.0
```

I don't have any other dependency in my test project, just "dropbox-stream".

Regards

**Comment by MohammedAl-Mahdawi** [comment]
I just created a test server on Linode and updated everything to the latest versions and the issue appear there.

If you want to take a look at the server just give me your email address or tell me how can I send the root server credentials to you and I'll do with my pleasure.

**Comment by velocityzen** [comment]
Fixed all in 1.1.0. Please update

**Comment by MohammedAl-Mahdawi** [comment]
@velocityzen Thank you so much, you are awesome!
